orttraining/orttraining/python/training/ortmodule/_logger.py

```diff
@@ -21,15 +21,18 @@
 
 class LogLevel(IntEnum):
     VERBOSE = 0
-    INFO = 1
+    DEVINFO = 1  # For ORT developers.
+    INFO = 1  # For ORT users.
```

Comment:
Looks when using the same value for `DEVINFO` and `INFO`, the value of `ORTMODULE_LOG_LEVEL_MAP[LogLevel.DEVINFO]` and `ORTMODULE_LOG_LEVEL_MAP[LogLevel.INFO]` are the same (`[<Severity.WARNING: 2>, 20]`). Is this expected?